ocs_ci/utility/utils.py

```diff
@@ -2433,3 +2433,20 @@ def configure_chrony_and_wait_for_machineconfig_status(
         # sleep here to start update machineconfigpool status
         time.sleep(60)
         wait_for_machineconfigpool_status(role)
+
+
+@retry(CommandFailed)
+def exec_cmd_on_debug_node(node, cmd):
+    """
+    Execute command on a node using debug pod
+
+    Args:
+        node (str): Node to create debug pod
+        cmd (str): Command to run on debug pod
+
+    Returns:
+        str: stdout of command
+
+    """
+    command = f"oc debug nodes/{node} -- {cmd}"
+    return run_cmd(command)
```

Comment:
Function `run_cmd(...)` is [deprecated](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/utils.py#L436-L438) in favor of [`exec_cmd(...)`](https://github.com/red-hat-storage/ocs-ci/blob/master/ocs_ci/utility/utils.py#L460-L514), so since this is a new code, I would propose to use `exec_cmd(...)` here.
```suggestion
    return exec_cmd(command)
```